tests/robustness/client.go

```diff
@@ -36,8 +36,8 @@ func NewClient(endpoints []string, ids identity.Provider, baseTime time.Time) (*
 	cc, err := clientv3.New(clientv3.Config{
 		Endpoints:            endpoints,
 		Logger:               zap.NewNop(),
-		DialKeepAliveTime:    1 * time.Millisecond,
-		DialKeepAliveTimeout: 5 * time.Millisecond,
+		DialKeepAliveTime:    10 * time.Second,
+		DialKeepAliveTimeout: 5 * time.Second,
```

Comment:
The reason of having so long keep alive timeouts was to make sure that client is quick to switch server after failpoint is injected. Traffic generation during robustness tests duration can be as low as 2s. During the test we inject failpoint that usually kills one of the servers. Clients connected to this server will need to quickly react so notice issue before 2s time out.

I think the current keep alive time is too low, but I would like it to be below 0.5s. What do you think?

Reply:
Sounds good. Will update it to 100ms.